tests/unit/test_vtk.py

```diff
@@ -197,6 +252,88 @@ def test_gb_2(self):
+
+        for g, d in gb:
+            pp.set_state(
+                d,
+                {
+                    "dummy_scalar": np.ones(g.num_cells) * g.dim,
+                    "dummy_vector": np.ones((3, g.num_cells)) * g.dim,
+                },
+            )
+
+        for e, d in gb.edges():
+            g = d["mortar_grid"]
+            pp.set_state(
+                d,
+                {
+                    "dummy_scalar": np.zeros(g.num_cells),
+                    "unique_dummy_scalar": np.zeros(g.num_cells),
+                },
+            )
+
+        subdomains_1d = gb.get_grids(lambda g: g.dim==1).tolist()
+        subdomains_2d = gb.get_grids(lambda g: g.dim==2).tolist()
+        g_2d = subdomains_2d[0]
+        interfaces = [e for e, d in gb.edges() if d["mortar_grid"].dim == 1]
+
+        save = pp.Exporter(
+            gb,
+            self.file_name,
+            self.folder,
+            binary=False,
+            export_constants_separately = False
+        )
+        save.write_vtu([
+            (subdomains_1d, "dummy_scalar"),
+            "dummy_vector",
+            "unique_dummy_scalar",
+            (g_2d, "cc", g_2d.cell_centers)
+        ])
+
+        with open(self.folder + self.file_name + "_1.vtu", "r") as content_file:
+            content = self.sliceout(content_file.read())
+        self.assertTrue(content == self._gb_3_grid_1_vtu())
+
+        with open(self.folder + self.file_name + "_2.vtu", "r") as content_file:
+            content = self.sliceout(content_file.read())
+        self.assertTrue(content == self._gb_3_grid_2_vtu())
+
+        with open(self.folder + self.file_name + "_mortar_1.vtu", "r") as content_file:
+            content = self.sliceout(content_file.read())
+        self.assertTrue(content == self._gb_3_mortar_grid_1_vtu())
+
+    def test_constant_data(self):
+        g = pp.StructuredTriangleGrid([3] * 2, [1] * 2)
+        g.compute_geometry()
+
+        dummy_scalar = np.ones(g.num_cells) * g.dim
+        dummy_vector = np.ones((3, g.num_cells)) * g.dim
+
+        save = pp.Exporter(
+            g,
+            self.file_name,
+            self.folder,
+            binary=False,
+        )
+        save.add_constant_data([(g, "cc", g.cell_centers)])
+        save.write_vtu([("dummy_scalar", dummy_scalar), ("dummy_vector", dummy_vector)])
+
+        with open(self.folder + self.file_name + "_2.vtu", "r") as content_file:
+            content = self.sliceout(content_file.read())
+        self.assertTrue(content == self._nonconstant_data_grid_vtu())
+
+        with open(self.folder + self.file_name + "_constant_2.vtu", "r") as content_file:
+            content = self.sliceout(content_file.read())
+        self.assertTrue(content == self._constant_data_grid_vtu())
+
+    # NOTE: This test is not directly testing Exporter, but the capability to export networks.
+    # Is this a used capability? If not, suggest removing the test and the corresponding capability.
```

Comment:
I think this can be useful when working with complex grids (difficult or expensive to mesh)